shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -2187,6 +2199,16 @@ - (void)triggerAutofillSave:(BOOL)saveEntries {
   [self addToInputParentViewIfNeeded:_activeView];
 }
 
+- (void)setPlatformViewTextInputClient:(long)platformViewID {
```

Comment:
Since we don't use the platformViewID, we can just remove it from the method signature?